.github/workflows/tests.yml

```diff
@@ -2,14 +2,22 @@ name: "libcyphal_test"
 
 on:
   push:
-    branches: [ "main" ]
+    branches:
+      - main
+      - 'issue/*'
+      # todo: temp - remove this later!
+      - 'sshirokov/CI_*'
   pull_request:
-    branches: [ "main" ]
+    branches:
+      - main
+      - 'issue/*'
+      # todo: temp - remove this later!
+      - 'sshirokov/CI_*'
```

Comment:
Isn't it about time to remove it now? You can add hashtag-based triggering like in CETL and elsewhere to trigger it at will.

Reply:
I'll do it soon.